app/services/PaymentAPIService.scala

```diff
@@ -57,6 +73,26 @@ class PaymentAPIService(wsClient: WSClient, paymentAPIUrl: String) {
       .execute()
   }
 
+  def getPaypalApiError(body: String): Option[PaymentApiError] = {
```

Comment:
use the `decode` function from circe to implement this method ([reference](https://circe.github.io/circe/codec.html))